### PR TITLE
- D3-976 Create Iroha account for DataCollector module

### PR DESCRIPTION
**Pull request by yurivin**
- D3-976 Create Iroha account for DataCollector module
- Client_accounts account

### CODE REVIEW

File: bootstrap/src/test/kotlin/jp/co/soramitsu/bootstrap/IrohaTest.kt [hunk 1]
```diff
@@ -9,6 +9,7 @@ import jp.co.soramitsu.bootstrap.genesis.d3.D3TestContext
 import jp.co.soramitsu.bootstrap.genesis.d3.D3TestGenesisFactory
 import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3
 import mu.KLogging
+import org.junit.Ignore
```
Comment (dolgopolovwork) [inline bootstrap/src/test/kotlin/jp/co/soramitsu/bootstrap/IrohaTest.kt]: I think we don't need it here

Comment (yurivin) [reply bootstrap/src/test/kotlin/jp/co/soramitsu/bootstrap/IrohaTest.kt]: done

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestContext.kt [hunk 2]
```diff
@@ -20,6 +20,11 @@ object D3TestContext {
             "notary",
             listOf("rmq")
         ),
+        AccountPrototype(
+            "data_collector",
```
Comment (Mingela) [inline bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestContext.kt]: where does this account come from?

**Review by Mingela — APPROVED** [review]

File: bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestContext.kt (hunk 2 above)
Comment (yurivin) [reply bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestContext.kt]: I creating this subsystem now

**Pull request merged**